src/System.Reflection.Metadata/src/System/Reflection/Internal/Utilities/FileStreamReadLightUp.cs

```diff
@@ -142,6 +142,9 @@ internal static unsafe bool TryReadFile(Stream stream, byte* buffer, long start,
             return true;
         }
 
+
+// The library guards against unavailable entrypoints by using EntryPointNotFoundException.
+#pragma warning disable BCL0015 // Diasable Pinvoke analyzer errors.
```

Comment:
Not sure, we might be able to get away by just encapsulating the pinvoke method declarations but since right now there were only two methods here, I went ahead and put it on the class itself.